modules/simple_away.cpp

```diff
@@ -211,7 +211,7 @@ class CSimpleAway : public CModule {
         if (sReason.empty()) sReason = SIMPLE_AWAY_DEFAULT_REASON;
 
         time_t iTime = time(nullptr);
-        CString sTime = CUtils::CTime(iTime, GetUser()->GetTimezone());
+        CString sTime = CUtils::CTime(iTime, "Etc/UTC") + " UTC";
```

Comment:
Is "Etc/UTC" different from "GMT"? Also `gmtime_r` would be cleaner than this.

Reply:
@DarthGandalf https://www.timeanddate.com/time/gmt-utc-time.html

> Although GMT and UTC share the same current time in practice, there is a basic difference between the two:
> - GMT is a time zone officially used in some European and African countries. The time can be displayed using both the 24-hour format (0 - 24) or the 12-hour format (1 - 12 am/pm).
> - UTC is not a time zone, but a time standard that is the basis for civil time and time zones worldwide. This means that no country or territory officially uses UTC as a local time.

Reply:
Etc/UTC is what the user would have specified in their GetTimezone() configuration.